### PR TITLE
Markdown conversion for `pt-BR` - Replace - Glossary section :warning:  Do not squash :warning: 

### PR DESCRIPTION
**Pull request by SphinxKnight**
Follow-up of https://github.com/mdn/translated-content/pull/8459

### CODE REVIEW

**Comment by SphinxKnight** [comment]
Please :warning: Do not squash merge :warning:

File: files/pt-br/glossary/idempotent/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+translation_of: Glossary/Idempotent
+original_slug: Glossario/Idempotente
+---
+Um método HTTP é **idempotente** se uma requisição idêntica pode ser feita uma ou mais vezes em sequência com o mesmo efeito enquanto deixa o servidor no mesmo estado. Em outras palavras, um método idempotente não deveria possuir nenhum efeito colateral (exceto para manter estatísticas). Implementados corretamente, o {{HTTPMethod("GET")}}, {{HTTPMethod("HEAD")}}, {{HTTPMethod("PUT")}}, e {{HTTPMethod("DELETE")}} são métodos **idempotentes**, mas não o método {{HTTPMethod("POST")}}. Todos os métodos {{glossary("seguro")}}s também são idempotentes.
+
+Para ser idempotente, somente o estado atual do _back-end_ de um servidor deve ser considerado, o código de status retornado por cada requisição pode variar: a primeira chamada de um {{HTTPMethod("DELETE")}} vai provavelmente retornar um {{HTTPStatus("200")}}, enquanto as chamadas sucessivas vão provavelmente retornar {{HTTPStatus("404")}}. Outra implicação do {{HTTPMethod("DELETE")}} ser idempotente é de que os desenvolvedores não deveriam implementar APIs RESTful com a\_ \_funcionalidade de deleção de última entrada usando o método `DELETE`.
```
Comment (josielrocha) [inline files/pt-br/glossary/idempotent/index.md:14]:
```suggestion
Para ser idempotente, somente o estado atual do _back-end_ de um servidor deve ser considerado, o código de status retornado por cada requisição pode variar: a primeira chamada de um {{HTTPMethod("DELETE")}} vai provavelmente retornar um {{HTTPStatus("200")}}, enquanto as chamadas sucessivas vão provavelmente retornar {{HTTPStatus("404")}}. Outra implicação do {{HTTPMethod("DELETE")}} ser idempotente é de que os desenvolvedores não deveriam implementar APIs RESTful com a funcionalidade de deleção de última entrada usando o método `DELETE`.
```

File: files/pt-br/glossary/accessibility/index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+title: Acessibilidade
+slug: Glossary/Accessibility
+tags:
+  - Acessibilidade
+  - Glossário
+translation_of: Glossary/Accessibility
+original_slug: Glossario/Acessibilidade
+---
+_Acessibilidade na Web (\_\_Web Accessibility)_ (**A11Y**) refere-se às melhores práticas para manter um site útil, apesar das restrições físicas e técnicas. A acessibilidade da Web é formalmente definida e discutida no {{Glossary("W3C")}} através de {{Glossary("WAI","Web Accessibility Initiative")}} (WAI).
```
Comment (josielrocha) [inline files/pt-br/glossary/accessibility/index.md:10]:
```suggestion
_Acessibilidade na Web (Web Accessibility)_ (**A11Y**) refere-se às melhores práticas para manter um site útil, apesar das restrições físicas e técnicas. A acessibilidade da Web é formalmente definida e discutida no {{Glossary("W3C")}} através de {{Glossary("WAI","Web Accessibility Initiative")}} (WAI).
```

**Review by josielrocha — APPROVED** [review]
Hi @SphinxKnight!

I found some empty `<em>`, but it was already present on HTML.

If would be nice to remove them before merge. But if you prefer remove them later I think we just need to create an issue for it.

What do you think?

**Comment by SphinxKnight** [comment]
> Hi @SphinxKnight!
> 
> I found some empty `\<em\>`, but it was already present on HTML.
> 
> If would be nice to remove them before merge. But if you prefer remove them later I think we just need to create an issue for it.
> 
> What do you think?

Well since this PR touches a large number of files, and that the issue may be on other files as well, I'll file an issue so that we can catch all occurrences afterwards.

**Comment by SphinxKnight** [comment]
N.B. I just pushed the conversion based on the latest main to resolve the conflict.

**Pull request merged**